app/controllers/events_controller.rb

```diff
@@ -7,7 +7,7 @@ class EventsController < ApplicationController
 
   # GET /events
   def index
-    @events = Event.sorted_by_start_date(!can?(:edit, Event)).reverse
+    @events = Event.sorted_by_start_date(!can?(:view_unpublished, Event))
```

Comment:
I was just wondering, because this is part of our user story #304 and I was afraid we would loose track if things are changed on the fly in PRs that have little to do with it.